build.gradle

```
@@ -57,3 +61,14 @@ sonarqube {
         property "sonar.host.url", "https://sonarcloud.io"
     }
 }
+
+spotless {
+    java {
+        target "**/*.java"
```

Spotless team recommends this as it includes less code unnecessarily:

```suggestion
        target "src/**/*.java"
```